ydb/public/sdk/cpp/client/iam/common/iam.h

```diff
@@ -168,7 +168,11 @@ class TGrpcIamCredentialsProvider : public ICredentialsProvider {
 
                     RequestInflight_ = false;
                     sleepDuration = std::min(BackoffTimeout_, BACKOFF_MAX);
-                    BackoffTimeout_ *= 2;
+                    if (BackoffTimeout_ < BACKOFF_MAX/2) {
+                        BackoffTimeout_ *= 2;
+                    } else {
+                        BackoffTimeout_ = BACKOFF_MAX;
+                    }
```

Comment:
```suggestion
                    BackoffTimeout_ = std::min(BackoffTimeout_ * 2, BACKOFF_MAX);
```
Так читаемее и единообразно со строкой выше

Reply:
там же переполнение на BackoffTimeout_ * 2 может быть
именно это и чиним